example.yml

```diff
@@ -141,6 +141,20 @@ metrics:
   aws_dimensions: [NodeID, ClusterIdentifier]
   aws_statistics: [Average]
 
+- aws_namespace: AWS/S3
+  aws_metric_name: BucketSizeBytes
+  aws_dimensions: [BucketName, StorageType]
+  aws_statistics: [Maximum]
```

Comment:
Only max?

Reply:
Changed to Average only, according to S3 Documentation: https://docs.aws.amazon.com/AmazonS3/latest/dev/cloudwatch-monitoring.html